src/SSHDebugPS/Connection.cs

```diff
@@ -126,6 +127,19 @@ internal string GetUserHomeDirectory()
             return _remoteSystem.FileSystem.GetDirectory(liblinux.IO.SpecialDirectory.Home).FullPath;
         }
 
+        public string AttachToProcess(int pid, string preAttachCommand)
+        {
+            var gdbStart = new liblinux.Services.GdbServerStartInfo();
+            gdbStart.ProcessId = pid;   // indicates an attach operation
+            gdbStart.PreLaunchCommand = preAttachCommand;
+            _gdbserver = _remoteSystem.Services.GdbServer.Start(gdbStart);
```

Comment:
>_gdbserver [](start = 12, length = 10)

Should we really be throwing if this comes back as null?

Reply:
Start does this, so removed the null test and doc'ed the behavior.